src/panel/poi/PoiCard.jsx

```diff
@@ -21,14 +21,16 @@ class PoiCard extends React.Component {
   }
 
   render() {
-    const { poi, closeAction, showDetails, openDirection } = this.props;
+    const { poi, closeAction, showDetails, openDirection, covid19Enabled } = this.props;
+
+    const hideOpeningHour = covid19Enabled && poi.blocks.find(b => b.type === 'covid19');
```

Comment:
`poi.blocks` may not exist here (for example for BragiPoi from geocoder results).
```suggestion
    const hideOpeningHour = covid19Enabled && poi.blocks && poi.blocks.find(b => b.type === 'covid19');
```

Reply:
Thank you!